### PR TITLE
fix 576

### PR DESCRIPTION
**Pull request by fabianfrz**
@DokuKaefer please check

### CODE REVIEW

File: src/www/diag_logs_settings.php [hunk 1]
```diff
@@ -399,7 +399,8 @@ function check_everything() {
                       <div class="hidden" for="help_for_logfilesize">
                         <?=gettext("Logs are held in constant-size circular log files. This field controls how large each log file is, and thus how many entries may exist inside the log. By default this is approximately 500KB per log file, and there are nearly 20 such log files.") ?>
                         <br /><br />
-                        <?=gettext("NOTE: Log sizes are changed the next time a log file is cleared or deleted. To immediately increase the size of the log files, you must first save the options to set the size, then clear all logs using the \"Reset Log Files\" option farther down this page. "); ?>
+                        <strong><span class="text-danger"><?=gettext("NOTE:");?></span></strong>
```
Comment (fichtner) [inline src/www/diag_logs_settings.php]: I neiher recommend the uppercase, nor the "danger/strong" combo for the word. If it's a note it's enough as it was (plus lower-case, if more severe than a note (warning or danger, the text "Warning:" or "Danger:" can be used.... but not a danger colour with a note...

File: src/www/system_authservers.php [hunk 2]
```diff
@@ -465,7 +465,8 @@ function select_clicked() {
                   <td>
                     <input name="ldap_host" type="text" id="ldap_host" size="20" value="<?=$pconfig['ldap_host'];?>"/>
                     <div class="hidden" for="help_for_ldap_host">
-                      <?= gettext("NOTE: When using SSL, this hostname MUST match the Common Name (CN) of the LDAP server's SSL Certificate."); ?>
+                      <span class="text-warning"><strong><?=gettext("Note:");?>&nbsp;</strong></span>
```
Comment (fichtner) [inline src/www/system_authservers.php]: Same here, is it a note or a warning?

File: src/www/diag_logs_settings.php (hunk 1 above)
Comment (fabianfrz) [inline src/www/diag_logs_settings.php]: This is what ticket #576 suggests. @DokuKaefer should answer this question.

Comment (DokuKaefer) [inline src/www/diag_logs_settings.php]: I found it under System - Networking
Several "Notes" are orange (see screenshot)
But the first "Note" at "Allow IPv6" is black.
Wouldn't it be better if all notes would be in orange?

networking_settings_system

Comment (fichtner) [inline src/www/diag_logs_settings.php]: I'd like no more colours, no more uppercase for those leading words. It leads to exaggerated exposure of minor notes (unless they are not minor, but then they should neither be notes nor explanation text above is conceptually correct hiding important details in further notes). I'm also against splitting up the "Note:" and the rest of the text, creating multiple gettext calls, it loses context for translation.

Comment (fabianfrz) [inline src/www/diag_logs_settings.php]: @fichtner @DokuKaefer close this pull request?

**Comment by fichtner** [comment]
Sorry I spent too many hours at the doctor's the past few days... @fabianfrz I'd like a fix to go in, but the patches should go into simplifying and unifying the code that we have. :)

**Pull request closed without merging**